metagraph/src/cli/query.cpp

```diff
@@ -811,6 +812,10 @@ int query_graph(Config *config) {
     assert(config->infbase_annotators.size() == 1);
 
     auto graph = load_critical_dbg(config->infbase);
+
+    if (config->canonical)
+        graph = std::make_shared<CanonicalDBG>(graph);
```

Comment:
do we need this if alignment is off?

Reply:
this is an old commit. now, it only does this if the underlying graph is not canonical

Reply:
ok, but could you check it if this is not going to make it slower for k-mer query without alignment?